### PR TITLE
Fix admin-panel styles

### PR DESCRIPTION
**Pull request by Dobriyr**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

Change px to rem
Fix hover on disabled blox
Remove width from tags
Fix map gap
Remove unused styles

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by ormykhalyshyn — APPROVED** [review]
nice job

**Review by MaksBrat — APPROVED** [review]

File: src/features/AdminPage/NewStreetcode/MainBlock/MainBlockAdmin.style.scss [hunk 1]
```diff
@@ -106,12 +73,14 @@
              }
         }
     }
+
     .textarea-teaser{
         height:f.pxToRem(80px);
     }
     .ant-input-textarea>textarea{
```
Comment (Katerix) [inline src/features/AdminPage/NewStreetcode/MainBlock/MainBlockAdmin.style.scss:80]: i would add some spaces around 'greater' symbol (>)

File: src/features/AdminPage/NewStreetcode/MapBlock/MapBlockAdmin.styles.scss [hunk 2]
```diff
@@ -2,11 +2,8 @@
 @use "@sass/_utils.functions.scss" as f;
 
 .mapBlockContainerr{
```
Comment (Katerix) [inline src/features/AdminPage/NewStreetcode/MapBlock/MapBlockAdmin.styles.scss:4]: meow

File: src/features/AdminPage/NewStreetcode/MapBlock/StatisticsToponymsAdmin/SearchMenu.styles.scss [hunk 3]
```diff
@@ -4,14 +4,10 @@
 @use "@sass/_utils.functions.scss" as f;
 
 .searchMenuToponyms {
-    margin: 10px;
+    gap: f.pxToRem(10px);;
```
Comment (Katerix) [inline src/features/AdminPage/NewStreetcode/MapBlock/StatisticsToponymsAdmin/SearchMenu.styles.scss:7]: double semicolon

File: src/features/AdminPage/NewStreetcode/MapBlock/StatisticsToponymsAdmin/StatisticsToponymsAdmin.styles.scss [hunk 4]
```diff
@@ -4,53 +4,20 @@
 @use "@sass/_utils.functions.scss" as f;
 
 .ToponymsTableWrapper {
+    display: flex;
+    flex-direction: column;
+    gap:10px;
```
Comment (Katerix) [inline src/features/AdminPage/NewStreetcode/MapBlock/StatisticsToponymsAdmin/StatisticsToponymsAdmin.styles.scss:9]: please add some spacing

**Pull request merged**